### ISSUE TITLE
Use current branch as default branch

### ISSUE THREAD

**Issue by krf** [oleander/git-fame-rb#37]
I like git-fame, but I noticed one big limitation right from the start:

```
$ git fame
/var/lib/gems/2.1.0/gems/git_fame-1.5.0/lib/git_fame/base.rb:185:in `populate': Does 'master' exist? (GitFame::BranchNotFound)
```

This is on 'dev' branch, on a repository which doesn't have a 'master' branch at all.

Now I can fix this by using `git fame -a dev`, but I'm surprised git fame just doesn't use the *current* branch as default, instead of 'master'. This would be just natural to me...

Tested with:  git-fame 1.5.0 (c) 2015 Linus Oleander

**Comment by oleander** [comment]
Yeah, this is a site effect of #29, which I'm planing to revert as it doesn't work as expected. Currently it falls back to `master` if no branch is defined. Give me a few days and I'll fix it.

Thanks for your report.

**Comment by oleander** [comment]
Fixed in version 1.7.0.
